themes-default/slim/src/components/helpers/config-textbox.vue

```diff
@@ -1,13 +1,14 @@
 <template>
-    <div id="config-textbox-content">
+    <div id="config-textbox">
```

Comment:
Keep in mind if this is uses more than once on the page you'll have two items with the same ID. I'd go with classes instead.

Reply:
Done